### PR TITLE
PEP8ified function.py

### PR DESCRIPTION
**Pull request by jremes-foss**
Hi all.

A bit lengthy pull request but this should do most of them for function.py.

There are still few to go (especially E501 in line 1048 is a bit tricky) but I'll handle that out as well.

I'll put this through CI just in case. 

Thanks.

### CODE REVIEW

**Comment by leosartaj** [comment]
Here's a quick output by flake8

```
flake8 sympy/core/function.py
sympy/core/function.py:92:16: E128 continuation line under-indented for visual indent
sympy/core/function.py:263:5: E129 visually indented line with same indent as next logical line
sympy/core/function.py:368:20: E128 continuation line under-indented for visual indent
sympy/core/function.py:603:17: W503 line break before binary operator
sympy/core/function.py:604:17: W503 line break before binary operator
sympy/core/function.py:1048:81: E501 line too long (117 > 80 characters)
sympy/core/function.py:1050:81: E501 line too long (83 > 80 characters)
sympy/core/function.py:1421:20: E128 continuation line under-indented for visual indent
sympy/core/function.py:1437:81: E501 line too long (84 > 80 characters)
sympy/core/function.py:1536:17: E128 continuation line under-indented for visual indent
sympy/core/function.py:1584:13: E128 continuation line under-indented for visual indent
sympy/core/function.py:1683:9: E128 continuation line under-indented for visual indent
sympy/core/function.py:2046:5: E128 continuation line under-indented for visual indent
sympy/core/function.py:2064:5: E128 continuation line under-indented for visual indent
sympy/core/function.py:2082:9: E128 continuation line under-indented for visual indent
sympy/core/function.py:2083:9: E128 continuation line under-indented for visual indent
sympy/core/function.py:2101:5: E128 continuation line under-indented for visual indent
sympy/core/function.py:2119:5: E128 continuation line under-indented for visual indent
sympy/core/function.py:2142:5: E128 continuation line under-indented for visual indent
sympy/core/function.py:2206:9: E128 continuation line under-indented for visual indent
sympy/core/function.py:2207:9: E128 continuation line under-indented for visual indent
sympy/core/function.py:2225:5: E128 continuation line under-indented for visual indent
sympy/core/function.py:2486:1: E402 module level import not at top of file
```
For this kind of a PR, it will be best to PEP8ify the whole file. Maybe you can look into these as well.

**Comment by Newman101** [comment]
Sure, I'll have a look.

File: sympy/core/function.py [hunk 1]
```diff
@@ -2346,7 +2355,8 @@ def count_ops(expr, visual=False):
                             ops.append(ADD)
                 if negs == len(aargs):  # -x - y = NEG + SUB
                     ops.append(NEG)
-                elif _coeff_isneg(aargs[0]):  # -x + y = SUB, but already recorded ADD
+                # -x + y = SUB, but already recorded ADD
```
Comment (gxyd) [inline sympy/core/function.py]: sometimes, in-line comments are useful. Rather than next-line comments.

**Comment by gxyd** [comment]
@Newman101 Can you please fix merge conflicts and address the requested changes?

File: sympy/core/function.py [hunk 2]
```diff
@@ -478,15 +480,16 @@ def bad(m):
...
                     m, n = m._mpc_
-                    return m[1] !=1 and m[-1] == 1 and \
-                        n[1] !=1 and n[-1] == 1
+                    return m[1] != 1 and m[-1] == 1 and \
+                        n[1] != 1 and n[-1] == 1
                 else:
                     return False
             if any(bad(a) for a in args):
-                raise ValueError  # one or more args failed to compute with significance
+                # one or more args failed to compute with significance
```
Comment (gxyd) [inline sympy/core/function.py]: don't change this.

**Comment by gxyd** [comment]
Most of the things are good as is. So I am -1 to this PR. I am closing this.

**Pull request closed without merging**